test/framework/src/main/java/org/opensearch/test/KeyStoreUtils.java

```diff
@@ -35,11 +37,17 @@ public static KeyStore createServerKeyStore() throws Exception {
     }
 
     private static X509Bundle generateCert() throws Exception {
-        return new CertificateBuilder().subject("CN=Test CA Certificate")
-            .setIsCertificateAuthority(true)
-            .algorithm(Algorithm.ed25519)
-            .provider(new BouncyCastleFipsProvider())
-            .buildSelfSigned();
+        final Locale locale = Locale.getDefault();
+        try {
+            Locale.setDefault(LocaleUtil.EN_Locale);
```

Comment:
The `LocaleUtil.EN_Locale` is used explicitly by `org.bouncycastle.asn1.x509.Time` (and very likely many other classes).